### ISSUE TITLE
Better Description Filtering

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Tantalus#12]
Being able to better filter through the descriptions would be helpful.  I like to look at one group at a time and it is hard to read them now.  But I don't know the best way to improve this.  Personally I would just swap out the Aggregation key for that, however I like how the aggregation Key is grouping them.

![image](https://user-images.githubusercontent.com/6305439/41874753-bf72dff4-7896-11e8-8f8a-2bf71173eb69.png)

**Comment by PRijnbeek** [comment]
Yes i see your point. Maybe you actually never want to look over description so we should make a separate table / selection field for it and remove it in the table view?

**Comment by ericaVoss** [comment]
Yeah, that could work.
